doc/language/globals.md

```diff
@@ -393,22 +477,44 @@ obtained by splitting `$_` by `$-F` is assigned at the start of each
 
 ### `$,`
 
+### `$;`
+
```

Comment:
This is an alias of `$-F`, not fully-deprecated.

Reply:
Fixed.